### PR TITLE
Convert rST double backticks to adoc single backticks

### PR DESCRIPTION
**Pull request by settermjd**
There were a number of instances where rST-style double-backticks still persisted. These are unnecessary in AsciiDoc.

### CODE REVIEW

File: modules/administration_manual/pages/configuration/files/external_storage/dropbox.adoc [hunk 1]
```diff
@@ -58,7 +58,7 @@ However, the internal Dropbox functionality should be removed in ownCloud 10.0.4
 
 Then, you need to provide a name for the folder in the "Folder name" field, and a "client key" and "client secret", located in the
 "Configuration" column.
-The client key and client secret values are the "App key" and ``App secret" fields which you saw earlier in your Dropbox app’s configuration settings page.
+The client key and client secret values are the "App key" and "App secret" fields which you saw earlier in your Dropbox app’s configuration settings page.
```
Comment (mmattel) [inline modules/administration_manual/pages/configuration/files/external_storage/dropbox.adoc]: shouldn´t hat be single backticks instead of quotes ?

**Comment by mmattel** [comment]
You have to reset config.sample back to master and remove it from this PR.
Then do the changes in core and create an own PR and let confg-to-docs run (always for both files).
config.sample files in docs will always be overwritten, you need to change the source in core...

File: modules/administration_manual/pages/configuration/files/external_storage/dropbox.adoc (hunk 1 above)
Comment (settermjd) [reply modules/administration_manual/pages/configuration/files/external_storage/dropbox.adoc]: Not in this case, as it has a trailing/wrapping double-quote.

**Comment by settermjd** [comment]
@mmattel, config.sample.php's been removed from this PR.

**Pull request merged**